### ISSUE TITLE
Browser's print preview dialog window is not displayed for a hidden browser window control

### ISSUE THREAD

**Issue by icnocop** [MicrosoftEdge/WebView2Feedback#3361]
**Description**

The browser's print preview dialog window is not displayed for a hidden browser window control.

It works as expected when using the system's print preview dialog.

**Version**

SDK: 1.0.1661.34
Runtime: Version 111.0.1661.62 (Official build) (64-bit)
Framework: WPF
OS: Windows 10 Enterprise x64 22H2 OS Build 19045.2728

**Repro Steps**

1. Add a hidden browser control to a WPF window:
`<wv2:WebView2 Visibility="Hidden" x:Name="browserForPrinting" />`

2. Try to display the browser's print preview dialog:
`browserForPrinting.CoreWebView2.ShowPrintUI(CoreWebView2PrintDialogKind.Browser);`

I expected the browser's print preview dialog to be displayed in a modal popup window.

It works as expected when using the system's print preview dialog:
`browserForPrinting.CoreWebView2.ShowPrintUI(CoreWebView2PrintDialogKind.System);`

![image](https://user-images.githubusercontent.com/507649/229943980-5c220c6f-5804-4888-824e-3c68e82aff3a.png)

Thank you.

**Comment by ajtruckle** [comment]
Yes. I had asked about this. I had to change my GUI logic to prevent using a hidden control. You will probably find seven tickets where I mentioned the same thing.

**Comment by ajtruckle** [comment]
https://github.com/MicrosoftEdge/WebView2Feedback/issues/3266

**Comment by Marim99** [comment]
I'm trying to print webview2 in UWP app when I use
_webView.CoreWebView2.ShowPrintUI(CoreWebView2PrintDialogKind.System);
it alert me with this message
"Check your printer or select another printer. The printer you chose isn't available or isn't installed correctly."

can you help me in this

**Comment by monica-ch** [comment]
@Marim99 Unfortunately, at the moment its a known limitation that printing in a UWP app is not supported. This is described a bit more in the [WebView2 WinUI2 doc](https://learn.microsoft.com/en-us/microsoft-edge/webview2/get-started/winui2#default-printing).

We are tracking this issue in https://github.com/MicrosoftEdge/WebView2Feedback/issues/3135. Feel free to add any additional feedback there. Thanks!
